csfieldguide/static/interactives/binary-cards/js/binary-cards.js

```diff
@@ -1,13 +1,14 @@
 var urlParameters = require('../../../js/third-party/url-parameters.js')
 "use strict";
-MAX_NUM_CARDS = 16;
+MAX_NUM_CARDS = 256;
 DEFAULT_NUM_CARDS_TO_SHOW = 8;
 
```

Comment:
(nowhere in particular) We seem to have lost dots for some cards and the total number of dots value is wrong. This is with the query string base=10

![image](https://user-images.githubusercontent.com/16066822/85983486-b58d6200-ba3b-11ea-866c-ce3374761fb8.png)

Reply:
:scream: I didn't check

Reply:
Yay for reviews!